packages/jest-config/src/Growler.js

```diff
@@ -0,0 +1,98 @@
+/**
+* Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+*
+* @flow
```

Comment:
Typically we put this below the license. Ignore me if you just copy-pasted from another file - we can do a bulk change later if that's the case.